homeworks/es6-basics/EvgeniyaP/main.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
There is no need in `index.html`. You can run this with Node.js or [jsbin](http://jsbin.com)